### PR TITLE
Update custom image for ocp4-anomaly-detection

### PR DESCRIPTION
**Pull request by chauhankaranraj**
We renamed the private repo for ocp4-anomaly-detection and created its public facing counterpart as per the guideline [here](https://github.com/aicoe-aiops/ai-enablement-initiative/blob/master/docs/create-consumable-artifacts.adoc#6-create-public-repository).

As a result, the image name also got updated. This PR addresses that name change.

### CODE REVIEW

**Review by tumido — DISMISSED** [review]
The new image is confirmed at https://registry-console.upshift.redhat.com/registry#/images/aicoe-notebooks/ocp4-anomaly-detection-internal

**Comment by chauhankaranraj** [comment]
Hey @rimolive could you PTAL and lmk if there's something else that needs to happen to merge this PR? :smiley:

**Comment by tumido** [comment]
@anishasthana @accorvin @HumairAK @durandom can we please get your :eyes: on this and unblock @chauhankaranraj ?

**Comment by durandom** [comment]
approved 👍

File: jupyterhub/bases/custom-images/jupyterhub-custom-images.yaml
Comment (rimolive) [inline jupyterhub/bases/custom-images/jupyterhub-custom-images.yaml:193]: Since the repo is internal you should avoid mention it publicly.

**Review by maulikjs — APPROVED** [review]

File: jupyterhub/bases/custom-images/jupyterhub-custom-images.yaml
Comment (rimolive) [inline jupyterhub/bases/custom-images/jupyterhub-custom-images.yaml:205]: Using `latest` tag is not recommended. I see there is another tag in the registry. Can we change to that?

**Review by rimolive — DISMISSED** [review]
I put some suggestions to it. Let me know if you have any questions.

File: jupyterhub/bases/custom-images/jupyterhub-custom-images.yaml
Comment (tumido) [reply jupyterhub/bases/custom-images/jupyterhub-custom-images.yaml:205]: Most of the images we import use `latest` tag. And others specify no tag, which also defaults to latest. See for yourself. It allows for a scheduled import.

https://github.com/AICoE/idh-manifests/blob/production/jupyterhub/bases/custom-images/jupyterhub-custom-images.yaml

I don't see a reason in locking this to a version on a live project, if an image stream update PR takes 2 months to be merged.

File: jupyterhub/bases/custom-images/jupyterhub-custom-images.yaml
Comment (tumido) [reply jupyterhub/bases/custom-images/jupyterhub-custom-images.yaml:193]: I don't think we care about this. We reference an internal image repository with the same content as is in this private repo. I don't see a reason why to obfuscate that. If this enables any attack vector in any way, we are doomed already for a long time.

**Review by HumairAK — APPROVED** [review]

**Comment by HumairAK** [comment]
@rimolive  please confirm if @tumido 's replies are sufficient, the request for changes is blocking the merge, and if you're satisfied I'd like to dismiss them.

File: jupyterhub/bases/custom-images/jupyterhub-custom-images.yaml
Comment (rimolive) [reply jupyterhub/bases/custom-images/jupyterhub-custom-images.yaml:205]: Lock to a specific tag other than latest will avoid break the image/notebook/whatever you are deploying. :)

If you are fine with this, I will be fine as well.

Comment (tumido) [reply jupyterhub/bases/custom-images/jupyterhub-custom-images.yaml:205]: > Lock to a specific tag other than latest will avoid break the image/notebook/whatever you are deploying

I don't see a way it can break even when the latest is applied. You spin up a pod with a certain image sha. That stays for the pod lifetime. Also there's a mapping between the source `DockerImage` tag to the `ImageStream` which should be respected. The `ImageStream` tag is named `latest` as well and is scheduled to be imported (every 15 mins by default). If the underlying image changes for example during pod scheduling, the pod is still scheduled with the old image sha since it doesn't get cleaned up by garbage collector because the image is being used.

File: jupyterhub/bases/custom-images/jupyterhub-custom-images.yaml
Comment (rimolive) [reply jupyterhub/bases/custom-images/jupyterhub-custom-images.yaml:193]: It depends. If there is sensitive data then would be good obfuscate or don't mention it publicly.

**Pull request merged**
